sigs/runtime-charter.md

```diff
@@ -0,0 +1,150 @@
+used in [modern cloud-native
+environments](https://github.com/cncf/toc/blob/master/DEFINITION.md)
+are in scope, including:
+
+
+
+*   generalized orchestration, autoscaling, scheduling, execution, container
+    runtimes, sandboxing, virtualization, image packaging and distribution and specialized architectures thereof, e.g. 
+    *   those aimed specifically at Edge, IoT, Batch, Big Data, AI/ML, etc
+    *   those incorporating specialized computing elements beyond CPUs, including GPUs, TPUs, FPGAs, ASICs, etc.
+
+We strive to understand the fundamental characteristics of different approaches with respect to availability,
+scalability, performance, consistency, ease-of-use, cost and operational complexity; and relate these to their
+suitability to various cloud-native use cases.
+
+
+## Areas Considered Out Of Scope
+
+Anything not considered in scope above is out of scope.  See also “Interfaces with Related Groups” below.
+
+Examples include:
+
+
+
+*   General authentication, authorization, accounting, auditing, etc (even though these clearly apply to )
+    - because AAA, etc is clearly the domain of the [CNCF Security SIG](https://github.com/cncf/sig-app-delivery).
+*   Getting network traffic into and out of workloads, or any of the interfaces related to that (e.g. CNI)
+    - these are the domain of the ([proposed](https://github.com/cncf/toc/blob/master/sigs/proposed.md)) CNCF Traffic SIG, and other related network-centric groups.  
+*   Local or remote storage associated with workloads
+     - this is the domain of the [CNCF Storage SIG](https://github.com/cncf/sig-storage) and related groups.
+
+
+## SIG Mission Statement
+
+To enable widespread and successful execution of the full spectrum of workload types,
+including both general latency-sensitive and batch, as well as more specialized
+categories listed as in scope, in cloud-native environments through:
+
+
+
+1. Providing valuable and unbiased information to the TOC,
+   End Users and Projects of the CNCF regarding areas considered in scope (see above).
+2. Collaborating effectively with other related groups (see below).
+3. Helping to maintain the continued health of the CNCF Projects deemed
+   to be within the scope of this SIG (see below)
+4. Identifying and filling gaps in the landscape of CNCF Projects within scope.
+
+Specific SIG deliverables are as per the above, and the [general SIG responsibilities
+set out by the CNCF TOC](https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md#responsibilities--empowerment-of-sigs).
+
+
+## Current CNCF Projects considered to be within the Scope of this SIG
+
+
+
+1. Kubernetes
+2. Containerd
+3. Harbor
+4. Dragonfly
+5. Virtual Kubelet
+6. CRI-O
+7. KubeEdge
+8. KubeVirt
+
+
+## Interfaces With Other Related Groups
+
+
+
+*   **[Several Kubernetes SIGs](https://github.com/kubernetes/community)**
+    cover Kubernetes-specific workload, scheduling, autoscaling, execution
```

Comment:
Can we add sig-node here?

Reply:
This item refers to all kubernetes sigs generally, including sig-node.  I don't think it's a good idea to enumerate them all explicitly, because there are very many of them, and they change over time.